stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.4.1
+version: 7.4.2
```

Comment:
Any time we add a new value, we bump a major version, so in this case: `7.5.0`

Reply:
fixed in [this commit](https://github.com/helm/charts/pull/23441/commits/a07c27a72b7226d5801e3de49fc80d16551a2bff)